treeherder/push_health/classification.py

```diff
@@ -1,3 +1,7 @@
+KNOWN_ISSUES = 'knownIssues'
+NEED_INVESTIGATION = 'needInvestigation'
```

Comment:
I find the addition of these confusing when looking at the code below.

Reply:
Line 70/71

Reply:
They're intended as constants for grouping the failures.  `KNOWN_ISSUES` used to be called 'intermittent' so that was overloaded.  Now, `KNOWN_ISSUES` can have more than just intermittents.

My intent was to use the equivalent of constants to make it less likely that one might cause a typo for one of these groupings.  

Is there a better way to make that less confusing?  Perhaps a comment for those two constants, at the very least, is in order.

Reply:
I think what's throwing me is that line 70/71 has the same variable names in all caps. Maybe I'm just tired :)

Reply:
Oh I see what you mean.  So those constant strings are used to define the keys for the `dict` called `classified`.  And then later, those same constants are used to reference those keys.  And that was my intention, to use those keys whenever referencing values in the `classified` dict.